recipe/meta.yaml

```diff
@@ -11,21 +11,21 @@ source:
     folder: opengl32sw                                                                                            # [win64]
 
   - url: https://anduin.linuxfromscratch.org/BLFS/qtwebengine/qtwebengine-{{ webengine_version }}.tar.xz
-  # - git_url: https://code.qt.io/cgit/qt/qtwebengine.git
-  #  git_rev: {{ webengine_version }}  # [not win]
     folder: qtwebengine
     patches:
       - patches/webengine/qt_webengine-3.15.9.patch  # [not win]
       - patches/webengine/0003-win.patch             # [win]
       - patches/webengine/0004-win8.patch            # [win]
+      - patches/webengine/qt_remove_override.patch
 
-  - git_url: https://code.qt.io/qt/qtwebengine-chromium.git  # [not win]
+  - git_url: https://github.com/qt/qtwebengine-chromium.git # [not win]
```

Comment:
Is there a reason why we are switching to the GitHub URL? This is a mirror, we would feel more comfortable referencing the original source trees, if possible. It doesn't matter for this particular release, but I just wanted to know if there's a reason why this change was made.

Reply:
You are right. Both urls in the above work but it is better to use the original source. This change propagated partly by mistake as I was doing some investigation to identify why chromium was giving me build-time problems (eventually I added qtweb_chromium_add_tag.patch to sort it out). I will change the url.